app/middlewares/auth.js

```diff
@@ -18,10 +19,18 @@ exports.authenticated = (request, response, next) => {
   if (token) {
     const { email } = sessionManager.decode(token);
```

Comment:
Are we sure this works correctly if we receive a token but its not valid, or cannot be decoded?